doc/client.md

```diff
@@ -1,75 +1,70 @@
-- [x]    `pool`        Query details of a liquidity pool
-- [x]    `pools`       Query for all liquidity pools
-- [x]    `swap`        Query for the swap message on the batch of the liquidity pool specified pool-id and msg-index
-- [x]    `swaps`       Query all swap messages in the liquidity pool batch
-- [x]    `withdraw`    Query the withdraw messages in the liquidity pool batch
-- [x]    `withdraws`   Query for all withdraw messages on the liquidity pool batch
-
-Progress REST/API
-
-- [x] liquidity query endpoints of REST api using grpc model
-- [x] broadcast txs using the new REST endpoint (via gRPC-gateway, beta1)
-
-
-## Tx
-
-`$ liquidityd tx liquidity --help`
+---
+title: Liquidityd 
+description: A high-level overview of how the command line (CLI) and REST API interfaces work for the liquidity module.
+---
+# Liquidityd 
+
+This document provides a high-level overview of how the command line (CLI) and REST API interfaces work for the liquidity module.
+
+## Transaction Command Line Interface
+
+- [MsgCreatePool](#msgcreatepool)
+  - Create liquidity pool
+- [MsgDepositWithinBatch](#msgdepositwithinbatch)
+  - Deposit to the liquidity pool batch
+- [MsgWithdrawWithinBatch](#msgwithdrawwithinbatch)
+  - Withdraw pool coin from the liquidity pool
+- [MsgSwapWithinBatch](#msgswapwithinbatch)
+  - Swap offer coin with demand coin from the liquidity pool with the given order price
+
+For error codes with the description, see [errors.go](https://github.com/tendermint/liquidity/blob/develop/x/liquidity/types/errors.go).
+
+## Query Command Line Interface
+
+- [Params](#params)
+  - Query the current liquidity parameters information
+- [Pool](#pool)
+  - Query details of a liquidity pool
+- [Pools](#pools)
+  - Query for all liquidity pools
+- [Batch](#batch)
+  - Query details of a liquidity pool batch
+- [Deposit](#deposit)
+  - Query for the deposit message on the batch of the liquidity pool
+- [Deposits](#deposits)
+  - Query for all deposit messages on the batch of the liquidity pool
+- [Withdraw](#withdraw)
+  - Query for the withdraw message on the batch of the liquidity pool
+- [Withdraws](#withdraws)
+  - Query for all withdraw messages on the batch of the liquidity pool
+- [Swap](#swap)
+  - Query for the swap message on the batch of the liquidity pool
+- [Swaps](#swaps)
+  - Query for all swap messages on the batch of the liquidity pool
+
+For error codes with the description, see [errors.go](https://github.com/tendermint/liquidity/blob/develop/x/liquidity/types/errors.go).
+
+## REST
+
+A node exposes the REST server default port of `1317`. Configure the port in `[api]` section of the `$HOME/.liquidityd/config/app.toml` file. When `swagger` param is set to `true`, you can check the Swagger documentation in your browser <http://localhost:1317/swagger-liquidity/>. You can also reference the Swagger documentation in the SwaggerHub [api-docs](https://app.swaggerhub.com/apis-docs/bharvest/).
```

Comment:
@kogisin the https://app.swaggerhub.com/apis-docs/bharvest/ link does not resolve for me. Tell me more about SwaggerHub api-docs? what is the target URL?

Reply:
@barriebyron This should be `https://app.swaggerhub.com/apis-docs/bharvest/cosmos-sdk_liquidity_module_rest_and_g_rpc_gateway_docs/`. I will fix this.